bundles/org.openhab.binding.upb/src/main/java/org/openhab/binding/upb/internal/message/UPBMessage.java

```diff
@@ -0,0 +1,220 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.upb.internal.message;
+
+import java.util.Arrays;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.smarthome.core.util.HexUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Model for a message sent or received from a UPB modem.
+ *
+ * @author cvanorman - Initial contribution
+ */
+@NonNullByDefault
+public class UPBMessage {
+
+    /**
+     * An enum of possible modem response types.
+     *
+     * @author cvanorman
```

Comment:
This author tag should be removed (probably a left over from the old binding)

Reply:
It is, this class was brought over from the old binding, so I'm uncomfortable removing the author tag.

Reply:
The author is already on the class so it's no problem removing the tag.